_rules/SC1-1-1-valid-duplicate-accessible-name-for-image.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Valid use of duplicate accessible names for images
+description: |
+	This rule checks that there is a valid reason for having a set of images with identical accessible names.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to any set of HTML or SVG element with the semantic role of `img`, in a document, exposed to assistive technologies, that have an identical [accessible name]().
+
+### Expectation
+
+Each test target has an identical purpose.
+
+**Note**: Leading and trailing whitespace and difference in case sensitivity should be ignored.
```

Comment:
Should this note be moved to "Applicability"?

Reply:
Its a standard accessibility support note that we have used on several rules relating specifically to AT support for the `tittle` attribute. I'm not sure what the point of the Accessibility Support note would be if not for this.

Reply:
Since it seems to relate to leading and trailing whitespace and case differences in the accessible names of applicable elements, it doesn't really make sense outside the "Applicability" section.

Reply:
I have moved the note on comparing accessible names into the definition for Accessible name: https://github.com/auto-wcag/auto-wcag/pull/254
We were using the note in at least 3 rules, so for consistency it should live in one place.